cmd/mayactl/app/command/volume_stats_test.go

```diff
@@ -65,6 +55,39 @@ func TestDisplayStats(t *testing.T) {
 				UsedLogicalBlocks:    "0",
 				WriteIOPS:            "0",
 			},
+			volume: v1alpha1.CASVolume{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						"vsm.openebs.io/controller-ips":    "10.48.1.17",
+						"vsm.openebs.io/iqn":               "iqn.2016-09.com.openebs.jiva:default-testclaimjiva",
+						"openebs.io/cluster-ips":           "10.51.242.184",
+						"openebs.io/iqn":                   "iqn.2016-09.com.openebs.jiva:default-testclaimjiva",
+						"openebs.io/replica-status":        "running, running, running",
+						"vsm.openebs.io/cluster-ips":       "10.51.242.184",
+						"vsm.openebs.io/replica-status":    "running, running, running",
+						"vsm.openebs.io/volume-size":       "5G",
+						"openebs.io/controller-ips":        "10.48.1.17",
+						"openebs.io/volume-size":           "5G",
+						"vsm.openebs.io/replica-count":     "3",
+						"vsm.openebs.io/replica-ips":       "10.48.0.7, 10.48.1.18, 10.48.2.7",
+						"openebs.io/controller-status":     "running,running",
+						"openebs.io/replica-count":         "3",
+						"vsm.openebs.io/controller-status": "running,running",
+						"openebs.io/replica-ips":           "10.48.0.7, 10.48.1.18, 10.48.2.7",
+						"openebs.io/targetportals":         "10.51.242.184:3260",
+						"vsm.openebs.io/targetportals":     "10.51.242.184:3260",
+					},
+				},
+				Spec: v1alpha1.CASVolumeSpec{
+					Capacity:     "5G",
+					CasType:      "jiva",
+					Iqn:          "iqn.2016-09.com.openebs.jiva:<no value>",
+					Replicas:     "1",
+					TargetIP:     "<no value>",
```

Comment:
Use some actual values in test.

Reply:
These values are generated from curl to m-apiserver